SolWatch/Game1.cs

```diff
@@ -26,11 +35,23 @@ public Game1()
         protected override void Initialize()
         {
             // TODO: Add your initialization logic here
-            solPosition = new Point(
+            screenCenter = new Point(
                 graphics.PreferredBackBufferWidth / 2,
                 graphics.PreferredBackBufferHeight / 2);
 
-            CelestialBodySymbolSize = new Point(50, 50);
+            celestialBodySymbolSize = new Point(50, 50);
+
+            maxOrbitRadiusInPixels = graphics.PreferredBackBufferHeight / 2;
+            maxOrbitSizeInPixels = new Point(maxOrbitRadiusInPixels * 2, maxOrbitRadiusInPixels * 2);
+
+            neptune = new Planet(
+                name: "Neptune",
+                semiMajorAxis: 4.5e9f,
+                longitudeOfAscendingNode: SolWatch.Utilities.RadiansFromDegrees(48.331f),
+                argumentOfPeriapsis: SolWatch.Utilities.RadiansFromDegrees(273.187f)
+                );
+
+            kmToPixelsMultiplier = ((maxOrbitRadiusInPixels * 2) / neptune.SemiMajorAxis);
```

Comment:
Why is this:
`maxOrbitRadiusInPixels * 2`
I would have thought this would cause twice as big as we want?